pkg/verifier/aws/entry_point.go

```diff
@@ -163,6 +163,7 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		"HTTP_PROXY":       vei.Proxy.HttpProxy,
 		"HTTPS_PROXY":      vei.Proxy.HttpsProxy,
 		"CACERT":           base64.StdEncoding.EncodeToString([]byte(vei.Proxy.Cacert)),
+		"NOPROXY":          vei.Proxy.NoProxyAsString(),
```

Comment:
```suggestion
		"NO_PROXY":          vei.Proxy.NoProxyAsString(),
```
nit: staying consistent with our other proxy-related metavars